commands/views.drush.inc

```diff
@@ -0,0 +1,526 @@
+<?php
+
+/**
+ * @file
+ * Drush integration for Views.
+ *
+ * Useful commands:
+ * - drush cache-clear views - Clears the views specific caches.
+ * - views-revert - Drush command to revert views overridden in the system.
+ */
+
+/**
+ * Implements hook_drush_help().
+ */
+function views_drush_help($section) {
+  switch ($section) {
+    case 'drush:views-revert':
+      $help = dt('Reverts views in the drupal installation that have been overriden. ');
+      $help .= dt('If no view names are specified, you will be presented with a list of overridden views to choose from. ');
+      $help .= dt('To revert all views, do not specify any view names, and choose the option "All" from the options presented.');
+      return $help;
+
+    case 'drush:views-list':
+      return dt('Show a list of available views with information about them.');
+
+    case 'drush:views-enable':
+      return dt('Enable the specified views. Follow the command with a space delimited list of view names');
+
+    case 'drush:views-disable':
+      return dt('Disable the specified views. Follow the command with a space delimited list of view names');
+  }
+}
+
+/**
+ * Implements hook_drush_command().
+ */
+function views_drush_command() {
+  $items = array();
+
+  $items['views-revert'] = array(
+    'description' => 'Revert overridden views to their default state. Make sure to backup first.',
+    'aliases' => array('vr'),
+    'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_FULL,
+    'callback' => 'views_revert_views',
+    'drupal dependencies' => array('views'),
```

Comment:
not sure if we have an analagous `backdrop_dependencies` for this, but in validation we can probably just check that `views` is enabled.